build.assets/version-check-latest/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"sort"
+	"time"
+
+	"golang.org/x/mod/semver"
+
+	"github.com/gravitational/trace"
+
+	go_github "github.com/google/go-github/v41/github"
```

Comment:
What's wrong with just `github`?

Reply:
The name `github` conflicts with `type github interface` on 63.  This was something I inherited from #9236 without questioning it.  I'm open to other ideas -- please reopen if with a suggestion if you'd like to see it handled differently.